src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,6 +1,10 @@
 package mate.academy.service;
 
+import mate.academy.model.User;
+
 public class AuthenticationService {
+    private static final UserService userService = new UserService();
```

Comment:
we want userService to be created individually for each new `AuthenticationService` object, not one for all
```suggestion
    private final UserService userService = new UserService();
```

Reply:
Sorry, 
fixed ...!